README.md

```diff
@@ -29,7 +29,7 @@
 </div>
 
 ## What is LASSL
-LASSL is a **LA**nguage library for **S**elf-**S**upervised **L**earning. LASSL aims to provide a easy-to-use framework for pretraining language model by only using Huggingface's Transformers and Datasets.
+LASSL is a **LA**nguage library for **S**elf-**S**upervised **L**earning. LASSL aims to provide an easy-to-use framework for pretraining language model by only using Huggingface's Transformers and Datasets.
```

Comment:
```suggestion
LASSL is a **LA**nguage framework for **S**elf-**S**upervised **L**earning. LASSL aims to provide an easy-to-use framework for pretraining language model by only using Huggingface's Transformers and Datasets.
```

Reply:
I think that we made a framewokr not library. Can we change this? @bzantium 

ref: https://webclub.tistory.com/458

Reply:
Thanks for good reference, I learned the difference of Framework, Library